automatic/poi/poi.nuspec

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <!-- == PACKAGE SPECIFIC SECTION == -->
+    <id>poi</id>
+    <version>9.0.0</version>
+    <owners>chocolatey, POI Contributors</owners>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/poi</packageSourceUrl>
+    <!-- ============================== -->
+
+    <!-- == SOFTWARE SPECIFIC SECTION == -->
+    <!-- This section is about the software itself -->
+    <title>poi</title>
+    <authors>POI Contributors</authors>
+    <projectUrl>https://poi.io</projectUrl>
+    <iconUrl>https://cdn.rawgit.com/poooi/poi/master/assets/icons/poi.png</iconUrl>
```

Comment:
You also need to add this icon the icons directory in this repository, you don't need to update this element though, just add the icon to the icons directory and a script of ours will take care of changing it when the package is updated.

Reply:
Fixed in 5be34ca